todos/009-complete-p2-inconsistent-command-format.md

```
@@ -0,0 +1,83 @@
+| Format | Locations | Examples |
+|--------|-----------|----------|
+| `/burlaki-*` (slash) | Lines 39-42, 114-118, 158, 168, 326-329 | `/burlaki-start`, `/burlaki-learn` |
+| `burlaki <cmd>` (space) | Lines 132, 138, 141, 461-462, 509 | `burlaki status`, `burlaki gate` |
+
+**Issue:**
+- No documented rule for when to use which format
+- Confusing for readers and implementers
+
+## Proposed Solutions
+
+### Option 1: Document convention (Recommended)
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)